src/ixdat/exporters/csv_exporter.py

```diff
@@ -42,16 +52,18 @@ def export_measurement(self, measurement, path_to_file, v_list=None, tspan=None)
             "".join([s_name + self.delim for s_name in s_list])[: -len(self.delim)]
             + "\n"
         )
+        #  TODO: Header may have to indicate which time goes with which value.
 
         lines = [header_line]
         max_length = max([len(data) for data in columns_data.values()])
         for n in range(max_length):
             line = ""
             for s_name in s_list:
                 if len(columns_data[s_name]) > n:
```

Comment:
I don't really understand what goes on here. n is in range(max_lenght), and therefore by definition < max_length, but max_lenght is the max len of datas in columns_data.values, then how could the length of any value in columns_data be > n?

Reply:
Comments added explaining.